core/src/test/scala/integration/kafka/server/DynamicBrokerReconfigurationTest.scala

```diff
@@ -64,6 +66,9 @@ import scala.collection.Seq
 object DynamicBrokerReconfigurationTest {
   val SecureInternal = "INTERNAL"
   val SecureExternal = "EXTERNAL"
+
+  val SslKeyStoreProps = Set(SSL_KEYSTORE_LOCATION_CONFIG, SSL_KEYSTORE_TYPE_CONFIG, SSL_KEYSTORE_PASSWORD_CONFIG, SSL_KEY_PASSWORD_CONFIG)
```

Comment:
nit: Maybe we can add a static set for the keystore and truststore configs to `SslConfigs`? We could probably then rewrite the new methods in `CertStores` to use them.

Reply:
Since `SslConfigs` is a public interface, I added a static to `CertStores` and changed this test to use that.